src/rpc/rawtransaction.cpp

```diff
@@ -819,7 +819,9 @@ static RPCHelpMan sendrawtransaction()
                 "\nNote that the transaction will be sent unconditionally to all peers, so using this\n"
                 "for manual rebroadcast may degrade privacy by leaking the transaction's origin, as\n"
                 "nodes will normally not rebroadcast non-wallet transactions already in their mempool.\n"
-                "\nAlso see createrawtransaction and signrawtransactionwithkey calls.\n",
+                "\nAlso see createrawtransaction and signrawtransactionwithkey calls.\n"
+                "\nNOTE: an RPC_TRANSACTION_ALREADY_IN_CHAIN exception will be raised if you are\n"
+                "attempting to send a transaction that already exists in a block.\n",
```

Comment:
suggestion to improve, tighten this up, and remove  "Note that" and "NOTE:"
```diff
+++ b/src/rpc/rawtransaction.cpp
@@ -816,12 +816,11 @@ static RPCHelpMan sendrawtransaction()
 {
     return RPCHelpMan{"sendrawtransaction",
                 "\nSubmit a raw transaction (serialized, hex-encoded) to local node and network.\n"
-                "\nNote that the transaction will be sent unconditionally to all peers, so using this\n"
+                "\nThe transaction will be sent unconditionally to all peers, so using sendrawtransaction\n"
                 "for manual rebroadcast may degrade privacy by leaking the transaction's origin, as\n"
                 "nodes will normally not rebroadcast non-wallet transactions already in their mempool.\n"
-                "\nAlso see createrawtransaction and signrawtransactionwithkey calls.\n"
-                "\nNOTE: an RPC_TRANSACTION_ALREADY_IN_CHAIN exception will be raised if you are\n"
-                "attempting to send a transaction that already exists in a block.\n",
+                "\nRPC_TRANSACTION_ALREADY_IN_CHAIN will be raised if the transaction already exists in a block.\n"
+                "\nRelated RPCs: createrawtransaction, signrawtransactionwithkey\n",
```

Reply:
which would change the help from
```
Submit a raw transaction (serialized, hex-encoded) to local node and network.

Note that the transaction will be sent unconditionally to all peers, so using this
for manual rebroadcast may degrade privacy by leaking the transaction's origin, as
nodes will normally not rebroadcast non-wallet transactions already in their mempool.

Also see createrawtransaction and signrawtransactionwithkey calls.

NOTE: an RPC_TRANSACTION_ALREADY_IN_CHAIN exception will be raised if you are
attempting to send a transaction that already exists in a block.
```
to
```
Submit a raw transaction (serialized, hex-encoded) to local node and network.

The transaction will be sent unconditionally to all peers, so using sendrawtransaction
for manual rebroadcast may degrade privacy by leaking the transaction's origin, as
nodes will normally not rebroadcast non-wallet transactions already in their mempool.

RPC_TRANSACTION_ALREADY_IN_CHAIN will be raised if the transaction already exists in a block.

Related RPCs: createrawtransaction, signrawtransactionwithkey
```

Reply:
Thanks, this is much better :)